integration/oci_calls/oci_call_test.sh

```diff
@@ -74,16 +83,15 @@ function setup() {
 }
 
 function run_oci_call() {
-	local -a oci_call=( "create" "start" "state" )
-
-	# This sleep is necessary to gather the correct logs
-	sleep 10
-
 	get_time
+	local -a oci_call=( "create" "start" "state" )
 
 	# Start a container
 	docker run -d --runtime=${RUNTIME} --name=${NAME} ${IMAGE} ${PAYLOAD}
 
+	cmd="docker ps -a --filter status=running"
```

Comment:
`sudo docker...`? 
also, should we specify the name of the container? In case there are other containers running?

Reply:
@chavafg changes applied